telemetry_package/main.py

```diff
@@ -1,33 +1,145 @@
-import sys
+import json
 import rclpy
-from rclpy.node import Node
+import socket
+import os
+import re
 
+from rcl_interfaces.msg import Log
 from common_package_py.common_node import CommonNode
+from rclpy.qos import QoSProfile, QoSReliabilityPolicy, QoSHistoryPolicy, QoSDurabilityPolicy, QoSLivelinessPolicy
 
+DEFAULT_UNIX_SOCKET_PATH = "/tmp/thi_drone"
```

Comment:
I think it would be a handy feature to pass the UNIX-socket-path using a program argument and fallback to this default if not provided. Just to make things a bit more _**### agile**_